apiary.apib

```diff
@@ -1801,6 +1801,36 @@ Get a list of all activity types.
                 + id: `9feb5f61-c6dd-4f9d-9b37-2813ada4e343` (string)
                 + name (string)
 
+# Group Todos
+
+## Todos [/todos]
+
+*Required scopes: `planning`*
+
+### todos.info [GET /todos.info]
+
+Get details for a single todo.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + id: `879770c5-8a7f-0067-8c25-b4993bdb5163` (string, required)
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + id: `879770c5-8a7f-0067-8c25-b4993bdb5163` (string)
+                + description: `Design the landing page` (string)
+                + activity_type (object)
+                    + type: `activityType` (string)
```

Comment:
This should be an actual activity type?

Reply:
I don't think so, in the future it will be possible to add your own 'activityType' which you can name whatever you want and can be identified with the uuid. You'd have to get the name of the activityType from the activityType end point